model/AliasHandler.php

```diff
@@ -441,6 +441,11 @@ public function delete() {
             return false;
         }
 
+        if ($this->can_delete) {
```

Comment:
Nice catch! Thanks for reporting it!

However I wonder if this condition is the wrong way round - I'd think we need
`    if ( ! $this->can_delete) {`
(note the added '!')